machinist/mserver/cmd/main.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"github.com/enfabrica/enkit/lib/kflags/kcobra"
+	"github.com/enfabrica/enkit/lib/oauth"
+	"github.com/spf13/cobra"
+)
+
+func Start(oauthFlags *oauth.RedirectorFlags) error {
+	return nil
```

Comment:
shouldn't this start an mserver object :)? I'd recommend using the 'lib/server' Run function, to set up the mux and actually run it.

Reply:
See above about CLI shape